<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parametrewindow</class>
 <widget class="QDialog" name="parametrewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paramètres</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../ressources/logo.png</normaloff>../ressources/logo.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>241</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QComboBox" name="choixDeviseComboBox">
      <item>
       <property name="text">
        <string>€</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>$</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>£</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CHF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>¥</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FRF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>§</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>₿</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="choixFormatDateComboBox">
      <item>
       <property name="text">
        <string>dd/mm/yy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>dddd dd mm yyyy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>dd/mm/yyyy</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Devise</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Format de date</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>10</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Paramètres</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
